Core/HLE/sceMp4.cpp

```diff
@@ -455,8 +465,15 @@ const HLEFunction sceAac[] = {
 	{0XD2DA2BBA, &WrapU_U<sceAacResetPlayPosition>,              "sceAacResetPlayPosition",           'x', "x"      },
 };
 
+const HLEFunction mp4msv[] = {
+	{0x3C2183C7, &WrapU_UU<mp4msv_3C2183C7>,                    "mp4msv_3C2183C7",               'x', "xx"      },
```

Comment:
Should these be removed from the sceMp4 lines, or hooked up there?

-[Unknown]

Reply:
Do nothing so that include in the file is better